Source/PH_Generic.cpp

```diff
@@ -1,4 +1,5 @@
 #include <cassert>
+#include "string.h"
```

Comment:
This should be `#include <cstring>` (angular brackets since this is a system include and `cstring` because it's the preferred way of including C headers in C++ code).